Dockerfile

```diff
@@ -57,7 +57,8 @@ RUN ARCH=$(uname -m); \
         gptfdisk \
         patterns-microos-selinux \
         btrfsprogs \
-	snapper \
+	    snapper \
```

Comment:
nit: there is something odd with spaces, I guess a mixture of tabs and spaces.

Reply:
Yep, something weird happened when resolving a merge conflict.. fixed!